### PR TITLE
56252 - pos bu and acl configuration

### PR DESCRIPTION
**Pull request by Vishal14583**
Configurable BU and ACL list added.
DevOps changes to update webapp appsetting

### CODE REVIEW

**Review by harshal11869 — APPROVED** [review]
Looks Good

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs [hunk 1]
```diff
@@ -285,12 +286,12 @@ private CreateBatchRequestModel CreateBatchRequestModel(Batch batchType)
...
             CreateBatchRequestModel createBatchRequest = new()
             {
-                BusinessUnit = "AVCSData",
+                BusinessUnit = _fssApiConfiguration.Value.BusinessUnit,
                 ExpiryDate = DateTime.UtcNow.AddDays(28).ToString("yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture),
                 Acl = new Acl()
                 {
-                    ReadUsers = new List<string>() { "public" },
-                    ReadGroups = new List<string>() { "public" }
+                    ReadUsers = string.IsNullOrEmpty(_fssApiConfiguration.Value.PosReadUsers) ? new() : _fssApiConfiguration.Value.PosReadUsers.Split(",").ToList(),
```
Comment (ajasonca) [inline UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs:293]: Is there a reason you're checking IsNullOrEmpty on the ReadUsers assignment, but you're not on the ReadGroups assignment?

**Review by ajasonca — CHANGES_REQUESTED** [review]
Just checking a slight inconsistency is correct...

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs (hunk 1 above)
Comment (Vishal14583) [reply UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs:293]: ReadGroups will always have value either public in prod or some group name in pre-prod testing.

**Review by ajasonca — APPROVED** [review]
Looks good thanks

**Pull request merged**